src/demo/App.js

```diff
@@ -29,8 +30,13 @@ class App extends Component {
     this.setState({ isCopied: true });
   };
 
+  handleChangeDisabled = (event) => {    
+    const hasDisabled = event.target.value;
+    this.setState({ disabled: hasDisabled });
+  };
```

Comment:
for checkboxes and radios we should use event.target.checked, not event.target.value